src/Cardano/AddressV3.cpp

```diff
@@ -213,4 +224,12 @@ Data AddressV3::data() const {
     return raw;
 }
 
+std::string AddressV3::getStakingAddress() const noexcept {
+    if (kind != Kind_Base || bytes.size() != (2 * HashSize)) {
+        return "";
+    }
+    const auto& stakingKeyHash = TW::subData(bytes, HashSize, HashSize);
+    return createReward(this->networkId, stakingKeyHash).string();
```

Comment:
Unify name? always use `Reward` address or `Staking` address

Reply:
Well, the address type names are taken from Cardano code, it is called a reward address type here, and it is used for staking. I put both names in the comment to optimize search: 'staking (reward) address'.